apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,20 @@
 # API Changelog
 
+
+## BFD-874 Fix diagnosis.sequence in V1 and V2
+
+ * Following FHIR mapping changes were made:
+  * Adds diagnosis.sequence if present in EOB.diagnosis, now, when a DRG is reported for V1 and V2, diagnosis.sequence is present in EOB.diagnosis. Previously, diagnosis.sequence was not present
```

Comment:
The formatting is off on this. I think the intention was to have an indentation but in rendered markdown this will just be two bullets. We probably don't need the second bullet.

Reply:
This is missing a period and reads a little funny. @bfausett77 can you suggest some verbiage for this change?